### ISSUE TITLE
Link RGI6 to RGI7

### ISSUE THREAD

**Issue by fmaussion** [GlacioHack/GlacierUtils#3]
As new versions of RGI will come out, it will become very useful to be able to link a gacier (system) in RGI6 to it's equivalent (if available) in RGI7.

We could use geopandas to create a look-up table linking RGI6 to RGI7. This can be tested with the GAMDAM inventory for example (since it will become RGI7)

**Comment by jlandmann** [comment]
Since it was my old task to do this kind if things, I am happy to take a look at this. However, we can discuss tomorrow morning.

**Comment by fmaussion** [comment]
Yes there are plenty of other projects, I just never came to describing them all. Let's chat tomorrow!
